allensdk/brain_observatory/ecephys/ecephys_session_api/ecephys_nwb_session_api.py

```diff
@@ -14,6 +14,25 @@
 
 color_triplet_re = re.compile(r"\[(-{0,1}\d*\.\d*,\s*)*(-{0,1}\d*\.\d*)\]")
 
+# TODO: If ecephys write_nwb is revisited, need to re-add `manual_structure_id`
+# column and add the structure ids to the nwbfile for the
+# add_ecephys_electrodes() function.
+STRUCTURE_ACRONYM_ID_MAP = {
```

Comment:
o_0. These are in warehouse. [OntologiesApi](https://github.com/AllenInstitute/AllenSDK/blob/master/allensdk/api/queries/ontologies_api.py) knows how to download obtain these records.

Reply:
These are in warehouse, but because stuff like https://github.com/AllenInstitute/AllenSDK/tree/master/allensdk/brain_observatory/ecephys/stimulus_analysis only load the nwbfile, (I think) that the `ecephys_structure_id` acronym map is the more straightforward solution.

Reply:
hardcoding a few rows of the structures table into this file is certainly a straightforward solution! It also creates another source of truth for these data. 

What if you treated these as a default & override with a (lims or) warehouse-derived mapping when this nwb interface is constructed from a cache?

Reply:
I guess the reason I was fairly confident about using the mapping in this case is because I don't really expect the LIMS `structure_id` that we assign to a given structure acronym to be changing.

If it were actual structure assignments for a given electrode/channel then yeah, I would be much more worried...